1_crosswalk_fetch.yml

```diff
@@ -100,13 +100,13 @@ targets:
 
   1_crosswalk_fetch/out/wqp_lake_temperature_sites_sf.rds.ind:
     command: fetch_wqp_lake_sites(target_name, characteristicName = wqp_temperature_charnames,
-      bBox = I(c(-105.79725, 34.95527, -78.85268, 50.12992)), dummy = I('2020-01-13'))
+      bBox = I(c(-105.79725, 34.95527, -74, 50.12992)), dummy = I('2020-01-13'))
```

Comment:
Good point. I don't like this function right now as it is a hack and the prior one did indeed need the dummy. When WQP stopped being able to handle a sites query w/o a spatial filter (like bbox), this function got more brittle, because the arguments now need to be updated when you modify your spatial area of interest (e.g., add NY). I will update the dummy and make the other change, but would like to come up with a generic way of getting all US sites in a mosaic pull. To answer your question below, no, the site query doesn't take too long.